pkgs/development/tools/misc/texinfo/6.0.nix

```diff
@@ -9,8 +9,11 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ perl xz ]
+    ++ stdenv.lib.optional stdenv.isSunOS libiconv
     ++ stdenv.lib.optional interactive ncurses;
 
+  configureFlags = "AWK=${gawk}/bin/awk";
```

Comment:
Aha, this file is new to me.
stdenv/native does not use it, stdenv/nix does.  Illumos uses stdenv/nix.

In this case, because /usr/bin is still accessible, the configure script detects the wrong awk first.
Maybe AWK should then be supplied like bash to every configure, just like SHELL is supplied to every call to make: https://github.com/NixOS/nixpkgs/commit/10ab227126bfab4ad1675b6b95f8c1e7e6370b2d


On 17 nov. 2015, at 16:27, Vladimír Čunát <notifications@github.com> wrote:

> In pkgs/development/tools/misc/texinfo/6.0.nix:
> 
> >      ++ stdenv.lib.optional interactive ncurses;
> >  
> > +  configureFlags = "AWK=${gawk}/bin/awk";
> Well, our stdenv is assumed to always provide (GNU) awk. pkgs/stdenv/common-path.nix contains a list of such packages. I believe the best approach is for all platforms to put GNU awk into stdenv, so we have better consistency.
> 
> —
> Reply to this email directly or view it on GitHub.
>